src/js/hca/config/form.js

```diff
@@ -25,6 +25,7 @@ import {
 } from '../helpers';
 
 import IntroductionPage from '../components/IntroductionPage';
+// import IntroductionPage from '../components/SIPIntroductionPage';
```

Comment:
I didn't realize I added this in this commit...But yeah, it's supposed to be `IntroductionPage` and not `SIPIntroductionPage` for now.